pre-docs/inventory_version2.md

```diff
@@ -75,8 +75,14 @@ currently Bolt only ships builtin lookup plugins. The plugins are:
 
 #### PuppetDB
 
-The PuppetDB plugin takes a `query` field, which is either a string containing
-a PQL query or an array containing a PuppetDB AST format query.
+The PuppetDB plugin takes a `query` field, which is either a string
```

Comment:
I think this needs to be more referency and may need more examples. I don't think I could use it based on this description.

perhaps breaking it up into two sections one about making a query to select the nodes and a second about setting "config/name/uri" based on facts and certname.